Project.toml

```diff
@@ -10,13 +10,15 @@ Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 GraphPlot = "a2cc645c-3eea-5389-862e-a155d0052231"
 Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
```

Comment:
I'd like to start moving back to JSON because JSON2 doesn't check order of the inputs, JSON is more robust.

Reply:
I think  you know about it, but just in case: https://github.com/quinnj/JSON3.jl

Reply:
Yep, wanted to try that, next weekend the plan is serialization/deserialization optimization - hoping JSON3 and StructTypes.jl would help with that.